packages/cloudflare/src/cli/build/index.ts

```diff
@@ -178,3 +180,79 @@ function ensureCloudflareConfig(config: OpenNextConfig) {
     );
   }
 }
+
+/**
+ * Creates a `wrangler.toml` file for the user if it doesn't exist, but only after asking for the user's confirmation.
+ *
+ * If the user refuses an error is thrown (since the file is mandatory).
+ *
+ * @param projectOpts The options for the project
+ */
+async function createWranglerTomlIfNotExistent(projectOpts: ProjectOptions): Promise<void> {
+  const wranglerTomlPath = join(projectOpts.sourceDir, "wrangler.toml");
```

Comment:
We should probably also check for .json or .jsonc.

Maybe we should create a .json file? It's probably easier for new comers.

Reply:
> We should probably also check for .json or .jsonc.

mhhh.... oh yeah that's a good call 🙂 

> Maybe we should create a .json file? It's probably easier for new comers.

I'm happy with whatever 👍 (we do recommend creating a toml file in the [docs](https://opennext.js.org/cloudflare/get-started#2-install-wrangler-and-add-a-wranglertoml-file))

Reply:
I think it's because json still required a flag when the docs were created.

Reply:
Updated 👍